src/main/java/core/basesyntax/Node.java

```diff
@@ -0,0 +1,39 @@
+package core.basesyntax;
+
+public class Node<K,V> {
+    private final int hash;
+    private final K key;
+    private V value;
+    private Node<K, V> next;
+
+    Node(int hash, K key, V value, Node<K, V> next) {
+        this.hash = hash;
+        this.key = key;
+        this.value = value;
+        this.next = next;
+    }
+
+    public final K getKey() {
+        return key;
+    }
+
+    public final V getValue() {
+        return value;
+    }
+
+    public int getHash() {
+        return hash;
+    }
+
+    public Node<K, V> getNext() {
+        return next;
+    }
+
+    public void setNext(Node<K, V> next) {
+        this.next = next;
+    }
+
+    public final void setValue(V newValue) {
+        value = newValue;
+    }
+}
```

Comment:
Don't use getters and setters in class Node. Since Node is an inner class, we have access to private fields of Node from the outer class, so using getters or setters is redundant. You can directly access the fields instead.